interface_config.js

```diff
@@ -14,8 +14,32 @@ var interfaceConfig = {
      *
```

Comment:
Auch hier die Frage: Da wir die Datei sowieso über Ansible verwalten, wollen wir hier Änderungen durchführen? Brauchst du das für die Entwicklung der Theme Anpassungen? Kann das ggf. in einen anderen Ort?